.gitignore

```diff
@@ -1,8 +1,6 @@
 /.rvmrc
 /.bundle
 /arturo*.gem
-/gemfiles/*.gemfile.lock
-/Gemfile.lock
```

Comment:
What is the reason for adding the lockfiles? Bundler 2 is already used (except for Rails 4.2) on CI.

Reply:
On second thought... I think we can close this PR. I only kept it alive with a rebase yesterday after being pinged about a review.
But tracking the lock files ensures that any platform, be it developer system or CI, will use the same dependency versions.